plugins/aws/aws.plugin.zsh

```diff
@@ -198,13 +198,32 @@ compctl -K _aws_profiles asp acp aws_change_access_key
 function aws_prompt_info() {
   local _aws_to_show
   local region="${AWS_REGION:-${AWS_DEFAULT_REGION:-$AWS_PROFILE_REGION}}"
+
   if [[ -n $AWS_PROFILE ]];then
-    _aws_to_show+="${ZSH_THEME_AWS_PROFILE_PREFIX:=<aws:}${AWS_PROFILE}${ZSH_THEME_AWS_PROFILE_SUFFIX:=>}"
```

Comment:
Just removing the `:` would do the trick. It should be `${ZSH_THEME_AWS_PROFILE_PREFIX=<aws:}`, for example.
There's no need of adding all those if cases.

Reply:
I didn't know this trick. Thank you! It was so hard to find [the reference](https://tldp.org/LDP/abs/html/parameter-substitution.html)...  I'll change it.